src/messages/search/MessageFlagsPredicate.cpp

```diff
@@ -42,6 +42,11 @@ MessageFlagsPredicate::MessageFlagsPredicate(const QString &flags)
         {
             this->flags_.set(MessageFlag::CheerMessage);
         }
+        else if (flag == "redemption")
+        {
+            this->flags_.set(MessageFlag::RedeemedChannelPointReward),
+                this->flags_.set(MessageFlag::RedeemedHighlight);
```

Comment:
Use a semicolon instead of a comma expression here:
```suggestion
            this->flags_.set(MessageFlag::RedeemedChannelPointReward);
            this->flags_.set(MessageFlag::RedeemedHighlight);
```